### ISSUE TITLE
Drag and drop not works with touch control

### ISSUE THREAD

**Issue by mmm286** [octgn/OCTGN#1806]
If I try to drag and drop a card from tablet without mouse when the card is placed in background not works

**Comment by kellyelton** [comment]
Yeah, you're unable to drag a card using your finger on the table.
